tests/convergence_tests/poisson_mms_common.rs

```diff
@@ -80,26 +84,49 @@ where
     // for linear contractions
     let u = DVector::<f64>::zeros(mesh.vertices().len());
 
-    let vector_assembler = SerialVectorAssembler::<f64>::default();
+    // Node coloring for test of parallel assembler
+    let colors = color_nodes(mesh);
+
+    let vector_assembler = VectorAssembler::<f64>::default();
     let matrix_assembler = CsrAssembler::default();
 
-    let laplace_assembler = ElementEllipticAssemblerBuilder::new()
+    let source_assembler = ElementSourceAssemblerBuilder::new()
         .with_finite_element_space(mesh)
-        .with_operator(&LaplaceOperator)
+        // TODO: Use better quadrature
         .with_quadrature_table(&quadrature)
-        .with_u(&u)
+        .with_source(poisson_source_function)
         .build();
 
-    let mut a_global = matrix_assembler.assemble(&laplace_assembler)?;
+    let mut b_global = vector_assembler.assemble_vector(&source_assembler)?;
 
-    let source_assembler = ElementSourceAssemblerBuilder::new()
+    // Compare with parallel assembler
+    {
+        let par_vector_assembler = VectorParAssembler::<f64>::default();
+        let mut par_b_global = DVector::zeros(b_global.len());
+        par_vector_assembler.assemble_vector_into(&mut par_b_global, &colors, &source_assembler)?;
+        assert_matrix_eq!(b_global, par_b_global, comp = float);
+    }
+
+    let laplace_assembler = ElementEllipticAssemblerBuilder::new()
         .with_finite_element_space(mesh)
-        // TODO: Use better quadrature
+        .with_operator(&LaplaceOperator)
         .with_quadrature_table(&quadrature)
-        .with_source(poisson_source_function)
+        .with_u(&u)
         .build();
 
-    let mut b_global = vector_assembler.assemble_vector(&source_assembler)?;
+    let mut a_global = matrix_assembler.assemble(&laplace_assembler)?;
+
+    // Compare with parallel assembler
+    {
+        let par_matrix_assembler = CsrParAssembler::default();
+        let pattern = par_matrix_assembler.assemble_pattern(&laplace_assembler);
+        let nnz = pattern.nnz();
+        let mut par_a_global = CsrMatrix::try_from_pattern_and_values(pattern, vec![0.0; nnz]).unwrap();
+        par_matrix_assembler
```

Comment:
`CsrAssembler` has a convenience function for also allocating the output storage. Would be nice to have there here too, as it would make the code here more compact (and it's generally useful when prototyping, before you start to weed out redundant allocations).